spec/credit_card_validations_spec.rb

```diff
@@ -105,48 +105,19 @@
 
   describe "adding/removing brand" do
 
-
-    let(:voyager_number) {
-      '869926275400212'
-    }
-
-    it "should validate number as voyager" do
-      CreditCardValidations::Detector.add_brand(:voyager, {length: 15, prefixes: '86'})
-      detector(voyager_number).valid?(:voyager).must_equal true
-      detector(voyager_number).voyager?.must_equal true
-      detector(voyager_number).brand.must_equal :voyager
-    end
-
-
-    describe "Add voyager rule" do
-      before do
-        CreditCardValidations::Detector.add_brand(:voyager, {length: 15, prefixes: '86'})
+    [:diners_us, :en_route, :laser, :voyager].each do |brand|
+      it "should support #{brand}" do
+        ->{ CreditCardValidations::Factory.random(brand) }.must_raise(CreditCardValidations::Error)
+        detector('somenumber').respond_to?("#{brand}?").must_equal false
+        require "credit_card_validations/plugins/#{brand}"
+        number = CreditCardValidations::Factory.random(brand)
+        CreditCardValidations::Detector.new(number).valid?("#{brand}".to_sym).must_equal true
+        detector('somenumber').respond_to?("#{brand}?").must_equal true
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.